### PR TITLE
Update dependencies - Closes #609

### LINKED ISSUE #609: Update dependencies on 2.0 version
### Description

Since the branch is living longer than expected, the dependency should be updated.

### PR DESCRIPTION
**Pull request by shuse2**
### Description
Update dependencies.

### Review checklist

* The PR solves #609 
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -90,23 +90,23 @@
...
-		"cli-table2": "LiskHQ/cli-table2#187c6ae80b9d963c598d0cb3379153387bfb15c4",
-		"inquirer": "6.0.0",
-		"lisk-elements": "1.0.0-rc.1",
-		"lockfile": "1.0.3",
-		"semver": "5.5.0",
+		"@oclif/command": "1.5.0",
+		"@oclif/config": "1.7.4",
+		"@oclif/errors": "1.2.0",
+		"@oclif/plugin-help": "2.1.1",
+		"babel-polyfill": "^6.26.0",
```
Comment (willclarktech) [inline package.json]: Why is this necessary?

File: package.json [hunk 2]
```diff
@@ -90,23 +90,23 @@
...
+		"bip39": "2.5.0",
+		"chalk": "2.4.1",
+		"cli-table2": "github:LiskHQ/cli-table2#187c6ae80b9d963c598d0cb3379153387bfb15c4",
```
Comment (willclarktech) [inline package.json]: What's the benefit of this? Just being explicit?

Comment (shuse2) [reply package.json]: This was automatically changed by npm
